tests/baselines/reference/convertToAsyncFunction/convertToAsyncFunction_CatchAndRej.ts

```diff
@@ -7,13 +7,15 @@ function /*[#|*/f/*|]*/():Promise<void> {
 
 async function f():Promise<void> {
     try {
+        let result: Response;
```

Comment:
I'd find it reassuring to know that @rbuckton is happy with this baseline change, which is probably easier than reviewing the implementation details.